integration_tests/test_string_01.py

```diff
@@ -0,0 +1,7 @@
+from string import ascii_lowercase, ascii_letters
+
+def test_string():
```

Comment:
You need to call the function below ;)

Reply:
ouch!

Reply:
fixed in #2788

Reply:
Perhaps one can write a custom flake8 plugin using https://pypi.org/project/flake8-plugin-utils/ that checks all functions that are defined, are being called.

Reply:
I don't think it has to be custom. You could probably use `ruff`, too :)

Reply:
LPython itself should give a warning for something like this.